js/index.js

```diff
@@ -1 +1,165 @@
-// Your code goes here
+// selectors
+const navLinks = document.getElementsByClassName('nav-link');
+const windowDivs = document.querySelectorAll('div');
+const [ contactForm ] = document.getElementsByClassName('contact-form');
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const footerText = document.querySelector('footer p');
+const arrowTop = document.getElementById('arrowTop');
+const buttons = document.querySelectorAll('.btn');
+
+// nav hover decoration
+Array.from(navLinks).forEach(nav => {
+  nav.addEventListener('click', (event) => event.preventDefault());
+  nav.addEventListener('mouseover', ({ target }) => {
+    target.style.textDecoration = 'underline';
+  });
+  nav.addEventListener('mouseout', ({ target }) => {
+    target.style.textDecoration = 'none';
+  });
+});
+
+// arrow key rotation
+(() => {
+  let x = 0;
+  let y = 0;
+  document.addEventListener('keydown', (event) => {
+    let fired;
+
+    switch (event.code) {
+      case 'ArrowUp':
+        x += 7;
+        fired = true;
+        break;
+      case 'ArrowDown':
+        x -= 7;
+        fired = true;
+        break;
+      case 'ArrowLeft':
+        y -= 7;
+        fired = true;
+        break;
+      case 'ArrowRight':
+        y += 7;
+        fired = true;
+        break;
+      default:
+        fired = false;
+        break;
+    }
+
+    if (fired) {
+      windowDivs.forEach(node => {
+        node.style.transform = `rotateX(${x}deg) rotateY(${y}deg)`;
+        event.preventDefault();
+      });
+    }
+  });
+})();
```

Comment:
very nice functionality with this, utilizing a switch statement and a creative transform animation